mods/tuxemon/l18n/en_US/LC_MESSAGES/base.po

```diff
@@ -7034,7 +7034,7 @@ msgstr "Tuxemon are the animals that inhabit this world.\n"
 "The strongest win, just like in the world of business.\n"
 "Most monsters are found in the wild. They morph into more advanced forms when they gain experience.\n"
 "But the five Pillars that make up the Cathedral, that is, the five allied companies that operate in this region,\n"
-"have improved on nature. This season, we offer five deluxe Tuxemon!"
+"Have improved on nature. This season, we offer five deluxe Tuxemon!"
```

Comment:
i think this was ok capitalized, since the previous line ended with a `,`, not `.`. there are a few others that didn't need to change, too